src/js/jsx/search/SearchBar.jsx

```diff
@@ -463,7 +479,7 @@ define(function (require, exports, module) {
             switch (event.key) {
                 case "Return":
                 case "Enter":
-                    if (selectedID === PLACEHOLDER_ID) {
+                    if (selectedID === PLACEHOLDER_ID || !selectedID) {
```

Comment:
If this disjunct is necessary, you should explain why in a comment. Also, these parameters are undocumented. JSDoc that explains what it means if `selectedID` is falsey would be a good idea.